.github/workflows/deploy_agent.yml

```diff
@@ -443,12 +448,23 @@ jobs:
       run_id: ${{ needs.get-release-info.outputs.workflow_run_id }}
     secrets: inherit
 
+  deploy-lambda-layer:
+    permissions:
+      contents: write
```

Comment:
Do we need to move this permission into the job that is ultimately called? Or do we not need write access?

Reply:
Yeah I think you're right, since the tag isn't getting created here.